SU-SWS/stanford_profile_helper
# D8CORE 2541 Added padding to images in the wysiwyg paragraph type

# READY FOR REVIEW

# Summary
- Add padding to images in the wysiwyg

# Need Review By (Date)
- 8/31

# Urgency
- medium

# Steps to Test
1. add an image to a wysiwyg
1. try different alignments and with & without captions
1. ensure the spacing below it is visible.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
